src/components/Schedule/SubmitButton.tsx

```diff
@@ -0,0 +1,96 @@
+import { gql, useMutation } from "@apollo/client";
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Button,
+  Flex,
+  useToast,
+} from "@chakra-ui/react";
+import { useEffect } from "react";
+import { UpdateScheduleInput } from "../../stores/schedule";
+
+const UPDATE = gql`
+  mutation update($input: UpdateScheduleInput!) {
+    updateSchedule(input: $input) {
+      message
+      success
+    }
+  }
+`;
+
+interface SubmitButtonProps {
+  handleSubmit: () => UpdateScheduleInput;
+  active: boolean;
+  setActive: (active: boolean) => void;
+}
+
+export const SubmitButton = (props: SubmitButtonProps) => {
+  const { handleSubmit, active, setActive } = props;
+
+  const [generate, { data, loading, error }] = useMutation(UPDATE);
+
+  const toast = useToast();
+
+  useEffect(() => {
+    if (!loading) {
+      if (data && !error) {
+        if (data.updateSchedule.success) {
+          toast({
+            title: "Schedule Updated",
+            status: "success",
+            duration: 3000,
+            isClosable: true,
+          });
+        } else {
+          console.log(data);
+          toast({
+            title: "Failed to Update Schedule",
+            description: data.updateSchedule.message,
+            status: "error",
+            isClosable: true,
+          });
```

Comment:
Put a `duration: null` on any error modals so they don't disappear and the user has time to read what went wrong